env

```diff
@@ -34,11 +34,21 @@ SENSU_CLIENT_GRAPHITE_CHECK_HOST=graphite:8003
 # let's simulate warning
 SENSU_CLIENT_CHECKS_BASE_TEST_NAME=basetest
 SENSU_CLIENT_CHECKS_BASE_TEST_COMMAND=/bin/false
-SENSU_CLIENT_CHECKS_BASE_TEST_SUBSCRIBERS=all
+SENSU_CLIENT_CHECKS_BASE_TEST_SUBSCRIBERS="all"
```

Comment:
It has to be otherwise it barfs....

"timestamp":"2015-09-14T14:50:28.733776+0000","level":"warn","message":"config file must be valid json","file":"/etc/sensu/conf.d/checks_role.json","error":"lexical error: invalid char in json text.\n          true,       \"subscribers\": [ all ],        \"subdue\": {      \n                     (right here) ------^\n"}

It's an array in the json file. 

So defining multiples would be like this:

SENSU_CLIENT_CHECKS_ROLE_TEST_SUBSCRIBERS="prod","dev"

which ends up in the file like this:

      "subscribers": [ "prod","dev" ],